UnityProject/Assets/Scripts/Objects/ObjectContainer.cs

```diff
@@ -78,6 +78,8 @@ public class ObjectContainer : MonoBehaviour, IServerLifecycle
 		// stored contents and their positional offsets, if applicable
 		private readonly Dictionary<GameObject, Vector3> storedObjects = new Dictionary<GameObject, Vector3>();
 
+		public Dictionary<GameObject, Vector3> StoredObjects => storedObjects;
```

Comment:
I don't encourage the usage of this shortcut, please use `GetStoredObjects()` instead.

Reply:
done